### PR TITLE
Create ECR for legal-framework-api

### PR DESCRIPTION
**Pull request by stephenrichards**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/legal-framework-api-uat/resources/ecr.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=4.4"
```
Comment (jasonBirchall) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/legal-framework-api-uat/resources/ecr.tf:2]: Any chance you can bump to latest `4.5`?

https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases

**Review by jasonBirchall — CHANGES_REQUESTED** [review]
Looks fine, just need to bump to the latest ECR release.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/legal-framework-api-uat/resources/ecr.tf (hunk 1 above)
Comment (stephenrichards) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/legal-framework-api-uat/resources/ecr.tf:2]: Will do, although this was auto generated by the cloud-platform CLI.

Comment (jasonBirchall) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/legal-framework-api-uat/resources/ecr.tf]: Ah, apologies for that. I'll get the template changed. :+1:

**Review by jasonBirchall — APPROVED** [review]

**Pull request merged**